readme.md

```diff
@@ -319,28 +319,31 @@ This is `undefined` if either:
 - the [`all` option](#all-2) is `false` (the default value)
 - both [`stdout`](#stdout-1) and [`stderr`](#stderr-1) options are set to [`'inherit'`, `'ipc'`, `'ignore'`, `Stream` or `integer`](https://nodejs.org/api/child_process.html#child_process_options_stdio)
 
-#### pipeStdout(target)
+#### pipeStdout(execaChildProcess)
 
-[Pipe](https://nodejs.org/api/stream.html#readablepipedestination-options) the child process's `stdout` to `target`, which can be:
-- Another [`execa()` return value](#pipe-multiple-processes)
-- A [writable stream](#save-and-pipe-output-from-a-child-process)
-- A [file path string](#redirect-output-to-a-file)
+`execaChildProcess`: [`execa()` return value](#pipe-multiple-processes)
 
-If the `target` is another [`execa()` return value](#execacommandcommand-options), it is returned. Otherwise, the original `execa()` return value is returned. This allows chaining `pipeStdout()` then `await`ing the [final result](#childprocessresult).
+[Pipe](https://nodejs.org/api/stream.html#readablepipedestination-options) the child process' `stdout` to another Execa child process' `stdin`.
 
-The [`stdout` option](#stdout-1) must be kept as `pipe`, its default value.
+Returns `execaChildProcess`, which allows chaining `pipeStdout()` then `await`ing the [final result](#childprocessresult).
 
-#### pipeStderr(target)
+[`childProcess.stdout`](#stdout) must not be `undefined`.
```

Comment:
This line is more accurate than the previous one.
For example, `{stdout: {file: 'file.txt'}}` works because it uses `pipe` under the hood.